plugins/janus_streaming.c

```diff
@@ -575,6 +584,32 @@ void *janus_streaming_watchdog(void *data) {
 	return NULL;
 }
 
+static int janus_streaming_lookup_multicast_interface(const struct ifaddrs * ifas, const char * iface, janus_network_address * result) {
+	if(result) {
```

Comment:
Please return immediately if `result` is invalid, no need to add an indentation layer just for that.